admin/class-boldgrid-backup-admin-task.php

```diff
@@ -0,0 +1,292 @@
+<?php
+/**
+ * File: class-boldgrid-backup-admin-task.php
+ *
+ * @link  https://www.boldgrid.com
+ * @since SINCEVERSION
+ *
+ * @package    Boldgrid_Backup
+ * @subpackage Boldgrid_Backup/admin
+ * @copyright  BoldGrid
+ * @version    $Id$
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+/**
+ * Class: Boldgrid_Backup_Admin_Task
+ *
+ * Tasks are similar to the jobs queue, but not quite the same.
+ *
+ * @since SINCEVERSION
+ *
+ * @todo Look into merging tasks with the jobs queue.
+ */
+class Boldgrid_Backup_Admin_Task {
+	/**
+	 * A date format used in returning times.
+	 *
+	 * For example, 'c' in date( 'c', time() ).
+	 *
+	 * @since SINCEVERSION
+	 * @var string
+	 */
+	public $date_format;
+
+	/**
+	 * The time this task was completed.
+	 *
+	 * @since SINCEVERSION
+	 * @access private
+	 * @var int
+	 */
+	private $completed_at;
+
+	/**
+	 * The time this task was created.
+	 *
+	 * @since SINCEVERSION
+	 * @access private
+	 * @var int
+	 */
+	private $created_at;
+
+	/**
+	 * Misc data associated with this task.
+	 *
+	 * @since SINCEVERSION
+	 * @access private
+	 * @var array
+	 */
+	private $data;
+
+	/**
+	 * Our helper class.
+	 *
+	 * @since SINCEVERSION
+	 * @access private
+	 * @var Boldgrid_Backup_Admin_Task_Helper
+	 */
+	private $helper;
+
+	/**
+	 * The task id.
+	 *
+	 * @since SINCEVERSION
+	 * @access private
+	 * @var string
+	 */
+	private $id;
+
+	/**
+	 * The time this task was started.
+	 *
+	 * @since SINCEVERSION
+	 * @access private
+	 * @var int
+	 */
+	private $started_at;
+
+	/**
+	 * The task type.
+	 *
+	 * For example: backup
+	 *
+	 * @since SINCEVERSION
+	 * @access private
+	 * @var string
+	 */
+	private $type;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since SINCEVERSION
+	 */
+	public function __construct() {
+		$this->helper = new Boldgrid_Backup_Admin_Task_Helper();
+	}
+
+	/**
+	 * Mark this task as being complete.
+	 *
+	 * @since SINCEVERSION
+	 */
+	public function end() {
+		$this->completed_at = time();
+
+		$this->update();
+	}
+
+	/**
+	 * Get the properties of this class.
+	 *
+	 * @since SINCEVERSION
+	 *
+	 * @return array
+	 */
+	public function get() {
+		return [
```

Comment:
Arrays should be declared with long array syntax per WordPress coding standards
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#declaring-arrays

Reply:
See similar comment above.